src/test/mock/CustomAxiosMockAdapter.ts

```diff
@@ -0,0 +1,429 @@
+              language: 'en',
+              profile_photo_url:
+                'https://lh3.googleusercontent.com/a-/AOh14GjGiq-zM2kKTZrRS3OhLHCPv6dk_6meXjCGKTwdYA=s128-c0x00000000-cc-rp-mo-ba3',
+              rating: 4,
+              relative_time_description: '2 months ago',
+              text:
+                'Crispy, juicy and moist fried chicken. I can see that they are using the clean oil for making fried chicken. But, there are a bit of fried dough, not the chicken and the one piece is pretty big, hoping that it would be a just one bite size. Other than that, the staffs are friendly and fast ready for pick up. I definitely would like to order again later.',
+              time: 1606809446,
+            },
+            {
+              author_name: 'Chaewon Kim',
+              author_url:
+                'https://www.google.com/maps/contrib/105237780839206869607/reviews',
+              language: 'en',
+              profile_photo_url:
+                'https://lh3.googleusercontent.com/a-/AOh14GhiCelp8IY_vD7-x2wtRnX2XI_7mNVUPilyk147UQ=s128-c0x00000000-cc-rp-mo',
+              rating: 5,
+              relative_time_description: 'in the last week',
+              text:
+                'THE BEST CHICKEN EVER!!\ndefinitely gonna come here regularly\nEvery menu comes with so many side dishes.\nlove it :D',
+              time: 1613892187,
+            },
+            {
+              author_name: 'Darren Sutton',
+              author_url:
+                'https://www.google.com/maps/contrib/105476603778596274534/reviews',
+              language: 'en',
+              profile_photo_url:
+                'https://lh3.googleusercontent.com/-Jr0TmdEAAmQ/AAAAAAAAAAI/AAAAAAAAAAA/AMZuuclQ9DyDodvfkWHm-C21ODKGWJciPA/s128-c0x00000000-cc-rp-mo/photo.jpg',
+              rating: 5,
+              relative_time_description: 'a month ago',
+              text:
+                'The fried chicken was delicious. We ordered the family size half and half and the portion size was good. The dry garlic and regular were both great, the dipping sauce was tasty too. My only complaint is that the dry garlic flavour was a bit too sweet.',
+              time: 1609376113,
+            },
+          ],
+          types: [
+            'restaurant',
+            'meal_takeaway',
+            'food',
+            'point_of_interest',
+            'establishment',
+          ],
+          url: 'https://maps.google.com/?cid=77286563717231905',
+          user_ratings_total: 104,
+          utc_offset: -480,
+          vicinity: '7235 Canada Way, Burnaby',
+          website: 'https://www.ribandchicken.ca/',
+        },
+        status: 'OK',
+        error_message: '',
+      }
+      //endregion
+    );
+  }
+
+  /**
+   * Mock invalid request response for place Details
+   * * Note: Invalid Request responses are still status code 200
+   * */
+  mockInvalidRequestPlaceDetails(): void {
+    this.onGet(defaultUrl).reply(200, {
```

Comment:
I'm wondering for this response and the not found response, do they also get returned with status 200?

Reply:
Correct!